patches/minecraft/net/minecraft/world/level/block/entity/ShulkerBoxBlockEntity.java.patch

```diff
@@ -1,14 +1,23 @@
 --- a/net/minecraft/world/level/block/entity/ShulkerBoxBlockEntity.java
 +++ b/net/minecraft/world/level/block/entity/ShulkerBoxBlockEntity.java
-@@ -235,6 +_,11 @@
-       return this.f_59647_ == ShulkerBoxBlockEntity.AnimationStatus.CLOSED;
+@@ -211,7 +_,7 @@
+    }
+ 
+    public boolean m_7155_(int p_59663_, ItemStack p_59664_, @Nullable Direction p_59665_) {
+-      return !(Block.m_49814_(p_59664_.m_41720_()) instanceof ShulkerBoxBlock);
```

Comment:
Personal preference, instead of replacing make this an OR and add a comment that you did, it is easier to patch and maintain vanilla compatibility then.

Reply:
fair point, will do